src/scene/materials/standard-material.js

```diff
@@ -643,6 +645,22 @@ class StandardMaterial extends Material {
         return this;
     }
 
+    /**
+     * Sets a vertex shader attribute on a material.
+     *
+     * @param {string} name - The name of the parameter to set.
+     * @param {string} semantic - Semantic to map the vertex data. Must match with the semantic set on vertex stream
+     * of the mesh.
+     * @example
+     * mesh.setVertexStream(pc.SEMANTIC_ATTR0, offset, 3);
+     * material.setAttribute('offset', pc.SEMANTIC_ATTR0);
+     */
+    setAttribute(name, semantic) {
+        if (!this.userAttributes.has(semantic)) {
```

Comment:
I'd say remove the if, just set it .. that'd allow to override existing ones too.